tools/tertiary-analysis/scater/scater-calculate-cpm.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="scater-calculate-cpm" name="Scater calculate cpm" version="0.1.0">
+<tool id="scater-calculate-cpm" name="Scater calculate cpm" version="0.0.3">
```

Comment:
Should we change the version to match Seurat itself?

Reply:
Yes, will do.

Reply:
Done